helm/cloud-provider-azure/templates/cloud-provider-azure.yaml

```diff
@@ -277,10 +278,12 @@ spec:
             path: /etc/kubernetes
         - name: etc-ssl
```

Comment:
would you mind changing this name as well?

Reply:
Now that the path is configurable we could change the name key to "ssl-mount" (remember to change both instances)

Thanks!

Reply:
Done